api-spec.md

```diff
@@ -87,12 +88,10 @@ operation to create an Item. Any query against a single OAFeat collection endpoi
 the collection name in the `collections` query parameter.
 
 It is recommended for clients use POST for querying (if the STAC API supports it), especially when using the 
-`intersects` query parameter, for two reasons:
+`intersects` query parameter, for one main reason:
 
 1. In practice, the allowed size for an HTTP GET request is significantly less than that allowed for a POST request, 
```

Comment:
no need to number if there's just one reason

Reply:
Fair, the second reason was no longer valid. Will update.